warehouse/templates/base.html

```diff
@@ -95,7 +95,7 @@
 
     <a href="#content" class="skip-to-content">Skip to main content</a>
 
-    <section class="stick-to-top js-stick-to-top">
+    <section id="stick-to-top" class="stick-to-top js-stick-to-top">
```

Comment:
I'd prefer if this had a more meaningful identifier that didn't clash with the class name, maybe something like `sticky-notifications` or `sticky-notification-section`